### PR TITLE
Expand readme

### PR DESCRIPTION
**Pull request by random-person-001**

### CODE REVIEW

**Comment by random-person-001** [comment]
Looking for approval for the direction I went with this, and also feedback on some of the smaller bits. If the larger picture is ok, the tweaks needed are:
- [ ]  write out instructions on making a pr/commit in browser
- [ ] if we decide we should, talk about acceptable content more 
- [ ] if we decide we should, talk about attribution more/rephrase

**Comment by peytondmurray** [comment]
> write out instructions on making a pr/commit in browser
Also what makes an appropriate/useful commit message, keeping in mind that it will be displayed on each page

File: README.md [hunk 1]
```diff
@@ -1,10 +1,41 @@
...
+
+### The DIY Way
+
+You can also do that integration process yourself. At this GitHub, you'll open a pull request with your change, which will be approved by a maintainer and automatically made live on the site. You'll need to make an account on GitHub for this. Specifically, the process goes:
+
+- on the wiki page you wish to change, on the right sidebar, click `Edit on GitHub`
+- make your changes (tip: just above the top left of the editor area, there's a edit / preview switch. Remember this isn't a perfect preview but it's sometimes helpful)
+- near the top right is a big green `Commit changes...` to click when you're satisfied
+- The `Commit message` is important; extended description etc is not. The commit message is what will be shown when folks look through a page's history, so replace the default and just breifly describe what was changed, and put your name in parentheses afterward. You don't need to say the file / page name here.
+  - example: "wrote base article (yourname)" or "added section for jabberwalk tips (yourname)" or "fixed inaccuracy about Vorpal Blades (yourname)"
```
Comment (peytondmurray) [inline README.md:29]:
```suggestion
  - example: "wrote base article" or "added section for jabberwalk tips" or "fixed inaccuracy about Vorpal Blades"
```

No need to include `(yourname)` here, it'll be pulled from the git history

**Review by peytondmurray — APPROVED** [review]
One small change, otherwise looks good.

**Pull request merged**